pywwt/core.py

```diff
@@ -62,6 +62,24 @@ def _send_msg(self, **kwargs):
 
     # TODO: need to add more methods here.
 
+    def clear_annotations(self):
+        """
+        Clears all annotations from the current view.
+        """
+        return self._send_msg(event='clear_annotations')
+    
+    def get_dec(self):
+        """
+        Return declination for the current view.
+        """
+        return self._send_msg(event='get_dec')
+
+    def get_ra(self):
```

Comment:
Rather than have two separate methods for RA and Dec, it would be nice to have a single ``get_center`` method that returns an Astropy SkyCoord - could you change that?